.gitignore

```diff
@@ -13,7 +13,8 @@ DerivedData/
 .netrc
 fastlane/report.xml
 fastlane/test_output/
-fastlane/.env.secret
+.env.secret
```

Comment:
This ignores both `.env.secret` files: the one at root level and the existing one under `fastlane`.